build.gradle

```diff
@@ -27,7 +27,7 @@ buildscript {
     dependencies {
         classpath 'com.netflix.nebula:gradle-netflixoss-project-plugin:11.5.0'
         classpath 'com.netflix.nebula:nebula-dependency-recommender:11.+'
-        classpath 'io.mantisrx:mantis-gradle-plugin:1.2.6-rc.3'
+        classpath 'io.mantisrx:mantis-gradle-plugin:1.2.6-rc.4'
```

Comment:
nit: Pl consider using a release version instead of a candidate.

Reply:
yeha will switch once this is fully fixed. (it's an open pr on the other side).

Reply:
(currently blocked due to gradle plugin repo publish failure).